const
obj
=
{
toString
(
)
{
throw
'
custom
error
'
;
}
}
;
async
function
f
(
)
{
await
import
.
defer
(
obj
)
.
catch
(
error
=
>
{
assert
.
sameValue
(
error
'
custom
error
'
)
;
}
)
.
then
(
DONE
DONE
)
;
}
f
(
)
;
